plugins/scaffolder/src/components/fields/EntityPicker/EntityPicker.tsx

```diff
@@ -21,10 +21,15 @@ import {
 import { TextField } from '@material-ui/core';
 import FormControl from '@material-ui/core/FormControl';
 import Autocomplete from '@material-ui/lab/Autocomplete';
-import { FieldProps } from '@rjsf/core';
+import { FieldProps, UiSchema } from '@rjsf/core';
 import React from 'react';
 import { useAsync } from 'react-use';
 
+export const allowArbitraryValues = (uiSchema: UiSchema): boolean => {
+  const allow = uiSchema['ui:options']?.allowArbitraryValues;
+  return typeof allow === 'boolean' ? allow : true;
```

Comment:
I think maybe a cleaner way could be do do `uiSchema['ui:options']?.allowAribitraryValues ?? true`

Which would basically set to `true` if it's not set, and if it's `false` then it would stay `false`.

Reply:
Nice suggestion.

I pushed a commit with this change. I didn't do a `push -f` on this one, I'll fix the commit history when all is good.

With this change, it would be easy to inline this instead of having a separate function. Is it worth keeping the separate function for the sake of unit tests?